includes/apple-exporter/components/class-component.php

```diff
@@ -746,18 +746,8 @@ protected static function remote_file_exists( $node ) {
 			return false;
 		}
 
-		// Fork for method of retrieval if running on VIP.
-		if ( defined( 'WPCOM_IS_VIP_ENV' ) && WPCOM_IS_VIP_ENV ) {
-			$result = vip_safe_wp_remote_get( $path );
-		} else {
-			$result = wp_safe_remote_get( $path );
-		}
-
-		// Check the headers in case of an error.
-		return ( ! is_wp_error( $result )
-			&& ! empty( $result['response']['code'] )
-			&& $result['response']['code'] < 400
-		);
+		$headers = get_headers( $path );
+		return stripos( $headers[0], '200 OK' ) ? true : false;
```

Comment:
Let's make sure that headers has a `[0]` first:
```suggestion
		return ! empty( $headers[0] ) && false !== stripos( $headers[0], '200 OK' );
```